packages/files/file.pony

```diff
@@ -80,7 +80,15 @@ class File
   Operations on a file.
   """
   let path: FilePath
+    """
+    The filesystem path to this file.
+    """
+
   let writeable: Bool
+    """
+    `true` if this file is writable.
```

Comment:
This one seems to not be conveying any useful information beyond that which is contained in the line above it.

Maybe it's better to say something like "the underlying file descriptor was opened as writable?

Also, should we consider fixing the misspelling of this field name now that it's been noticed?

Reply:
I took a look around and it seems, both variants are in use, `writable` and `writeable`, although `writable` seems to be more common. (I am not a native english speaker, so correct me if i'm wrong).

I am a bit hesitant to introduce a breaking change like this into this PR that should only be about adding documentation.